multi_block_core.dic

```diff
@@ -241,6 +276,388 @@ save_exptl_crystal.id
+    _type.container               Single
+    _type.contents                Word
+
+save_
+
+save_SPACE_GROUP_SYMOP
+
+    _definition.id                SPACE_GROUP_SYMOP
+    _definition.scope             Category
+    _definition.class             Loop
+    _definition.update            2023-10-16
+    _description.text
+;
+    The CATEGORY of data items used to describe symmetry equivalent sites
+    in the crystal unit cell.
+;
+    _name.category_id             SPACE_GROUP
+    _name.object_id               SPACE_GROUP_SYMOP
+
+    loop_
+      _category_key.name
+         '_space_group_symop.id'
+         '_space_group_symop.space_group_id'
+
+save_
+
+save_space_group_symop.space_group_id
+
+    _definition.id                '_space_group_symop.space_group_id'
+    _definition.update            2023-01-17
+    _description.text
+;
+    A code which identifies the space group to which the symop belongs.
+;
+    _name.category_id             space_group_symop
+    _name.object_id               space_group_id
+    _name.linked_item_id          '_space_group.id'
+    _type.purpose                 Link
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Word
+
+save_
+
+save_SPACE_GROUP_WYCKOFF
+
+    _definition.id                SPACE_GROUP_WYCKOFF
+    _definition.scope             Category
+    _definition.class             Loop
+    _definition.update            2023-10-16
+    _description.text
+;
+    Contains information about Wyckoff positions of a space group.
+    Only one site can be given for each special position, but the
+    remainder can be generated by applying the symmetry operations
+    stored in _space_group_symop.operation_xyz.
+;
+    _name.category_id             SPACE_GROUP
+    _name.object_id               SPACE_GROUP_WYCKOFF
+
+    loop_
+      _category_key.name
+         '_space_group_Wyckoff.id'
+         '_space_group_Wyckoff.space_group_id'
+
+save_
+
+save_space_group_wyckoff.space_group_id
+
+    _definition.id                '_space_group_Wyckoff.space_group_id'
+    _definition.update            2023-01-17
+    _description.text
+;
+    A code which identifies the space group to which the Wyckoff position
+    belongs.
+;
+    _name.category_id             space_group_Wyckoff
+    _name.object_id               space_group_id
+    _name.linked_item_id          '_space_group.id'
+    _type.purpose                 Link
+    _type.source                  Assigned
+    _type.container               Single
+    _type.contents                Word
+
+save_
+
+save_STRUCTURE
+
+    _definition.id                STRUCTURE
+    _definition.scope             Category
+    _definition.class             Set
+    _definition.update            2023-10-16
+    _description.text
+;
+    The DICTIONARY group encompassing the CORE STRUCTURE data items defined
+    and used within the Crystallographic Information Framework (CIF).
```

Comment:
This category definition needs to be replaced. It is no longer a 'dictionary group' but just an ordinary category. Something like "A category for collecting information about a crystallographic structure"

Reply:
Done